handbook/python-dataplugins/python-dataplugins.md

```diff
@@ -285,12 +285,26 @@ export a DataPlugin as a URI file.
    <figcaption>Exporting DataPlugins in DIAdem</figcaption>
 </figure>
 
+You can also export a DataPlugin from NI's [Python DataPlugin Extension](https://github.com/ni/vscode-ni-python-dataplugins)
+for Visual Studio Code:
+
+<figure>
+   <img src="../../img/pydp-vscode_export.png" width="600" />
+   <figcaption>Exporting DataPlugins in VSCode</figcaption>
+</figure>
+
+### Encrypted DataPlugins
+
+In some cases, you may want to protect your code from being viewed by others.
+Use DIAdem to export an encrypted Python DataPlugin. This functionality is not

```

Comment:
Should this be "NI DIAdem"?

Reply:
I believe you can let DIAdem stay alone, see https://www.ni.com/en-us/shop/software/products/diadem.html
@GrKuehnH can you confirm?